src/behavior.ts

```diff
@@ -446,11 +436,11 @@ class SwitcherBehavior<A> extends ActiveBehavior<A>
     this.b = newB;
     this.parents = cons(newB);
     newB.addListener(this.bNode, t);
-    const newState = newB.state;
-    if (newState !== this.state) {
-      this.changeStateDown(newState);
+    this.changeStateDown(newB.state);
```

Comment:
Why is `changeStateDown` always called instead of only when `newState !== this.state` as before:

```ts
const newState = newB.state;
if (newState !== this.state) {
  this.changeStateDown(newB.state);
}
```

Reply:
because this is already being checked inside `this.changeStateDown`

Reply:
I see. Great change then :+1: